otelib/backends/python/base.py

```diff
@@ -133,7 +133,7 @@ def get(self, session_id: "Optional[str]" = None) -> bytes:
         """
         if session_id is None:
             session_id = f"session-{str(uuid4())}"
-            self.cache[session_id] = {}
+            self.cache[session_id] = AttrDict()
```

Comment:
It shouldn't matter whether it's an `AttrDict` or a `dict`. The purpose of the `AttrDict` is merely to wrap a standard Python `dict` in a pydantic `BaseModel`.
I.e., this change is not necessary, rendering the PR not necessary.